addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
@jkuri I'm okay with the task then.  For documentation, could you point to the issue on ember-cli about problems on Windows. Ideally add a link to it in a comment in the file describing why we do our own task.